### ISSUE TITLE
Java checker error stops java from running.

### ISSUE THREAD

**Issue by dshelstad** [MultiMC/MultiMC5#1679]
System Information 
MB Asus Z-170AR, Intel i7-6700, 4.0GH, 32 GB,  GTX 980ti
-----------------------------
MultiMC version:
0.4.11-751
Operating System:
Win 10 Anniversary (Home Ed)
Summary of the issue or suggestion:
----------------------------------------------
4.650 D "Running java checker: C:/Program Files/Java/jre1.8.0_101/bin/java.exe -jar C:/Users/dshelstad/Desktop/MultiMC/jars/JavaCheck.jar"
    4.785 D STDOUT "os.arch=amd64
java.version=1.8.0_101
[0x7FFDBD6870E3] ANOMALY: use of REX.w is meaningless (default operand size is 64)
"
    4.785 W STDERR ""
    4.785 D Java checker finished with status  0  exit code  0
    4.785 D Java checker failed - couldn't extract required information.
    4.786 C Task failed:  "Could not start java!"
    4.794 C Task failed:  "Could not start java!"

What should happen:
------------------------------
The stranded error is stopping MMC from running..

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1.Try running any instance.

2.Do a java check in the settings.

3.

Suspected cause:
---------------------------
Windows changed something in 
[MultiMC-0.log.txt](https://github.com/MultiMC/MultiMC5/files/509484/MultiMC-0.log.txt)

standard IO in the update stopping javachecker from functioning properly...
----------------------------
[//]: # (Please refer to https://github.com/MultiMC/MultiMC5/wiki/Log-Upload for instructions on how to attach your logs.)

Additional Info:
---------------------------
Thanks and good luck with this bloody update.

**Comment by peterix** [comment]
You have some crapware getting injected into every process on your system, and it's breaking things on a rather fundamental level.

Do you use AVG? If yes, update AVG and hope it's fixed.

See:
https://support.avg.com/answers?id=906b0000000D4GzAAK

There are other possible sources of this issue... I have no idea what other software you are using... you might have to do some investigation yourself.

**Comment by peterix** [comment]
Also:
https://social.technet.microsoft.com/Forums/office/en-US/efd83b6a-dc86-4854-b05f-f484e1ed5596/anomaly-meaningless-rex-prefix-used?forum=winserverGP

http://stackoverflow.com/questions/36873664/anomaly-meaningless-rex-prefix-used

And possibly more.

Basically, something extra gets loaded into your processes and prints extra garbage it has no business of printing to the terminal.

**Comment by peterix** [comment]
Could be display drivers...

**Comment by peterix** [comment]
And this:
https://community.spiceworks.com/topic/1788898-win-10-build-1607-causing-issues-with-trend-micro

**Comment by dshelstad** [comment]
I had bad update to Win 10 Anniversary 1 month ago where a bug was causing shutdown 20 secs using anything task bar related.  It was reported in the Windows forums as happening to quit a few people.  I had to fallback to the previous version.  Then Saturday I upgraded after I confirmed that Microsoft had addressed the prior issues. The update worked, But had to update  usb drivers and video drivers. The only problem I have is been with MMC not running?  P.S.  I am running Trend Micro as my Anti Virus.

Thanks for the reply's.
